tools/SetupFlow/DevHome.SetupFlow/Views/InstallationNotesView.xaml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Page
+    x:Class="DevHome.SetupFlow.Views.InstallationNotesView"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <Grid Name="FullPage">
+        <!-- Extended TitleBar -->
+        <Grid x:Name="AppTitleBar"
+              Canvas.ZIndex="0"
+              Height="30"
+              IsHitTestVisible="True"
+              VerticalAlignment="Top">
+            <Image 
+                Source="{x:Bind DevHomeIconPath, Mode=OneWay}"
+                HorizontalAlignment="Left"
+                Width="16"
+                Height="20" 
+                Margin="10 5 5 0"/>
+            <TextBlock x:Name="AppTitleBarText"
+                VerticalAlignment="Center"
+                TextWrapping="NoWrap"
```

Comment:
Set a TextTrimming in the unlikely but possible scenario that the app has as really long name?

Reply:
In this particular case, it is possible since the title includes the package name